[2025-06-04T09:16:06.424+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-04T09:16:06.449+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_postgres_create_table_dag.create_postgres_table manual__2025-06-04T09:16:05.437827+00:00 [queued]>
[2025-06-04T09:16:06.459+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_postgres_create_table_dag.create_postgres_table manual__2025-06-04T09:16:05.437827+00:00 [queued]>
[2025-06-04T09:16:06.459+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-04T09:16:06.471+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_postgres_table> on 2025-06-04 09:16:05.437827+00:00
[2025-06-04T09:16:06.475+0000] {standard_task_runner.py:63} INFO - Started process 32 to run task
[2025-06-04T09:16:06.478+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_postgres_create_table_dag', 'create_postgres_table', 'manual__2025-06-04T09:16:05.437827+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/test_postgres_conn_and_create_table.py', '--cfg-path', '/tmp/tmp04g_k6fq']
[2025-06-04T09:16:06.479+0000] {standard_task_runner.py:91} INFO - Job 137: Subtask create_postgres_table
[2025-06-04T09:16:06.526+0000] {task_command.py:426} INFO - Running <TaskInstance: test_postgres_create_table_dag.create_postgres_table manual__2025-06-04T09:16:05.437827+00:00 [running]> on host 8b60c3c987d2
[2025-06-04T09:16:06.600+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_postgres_create_table_dag' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-04T09:16:05.437827+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-04T09:16:05.437827+00:00'
[2025-06-04T09:16:06.601+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-04T09:16:06.628+0000] {base.py:84} INFO - Using connection ID 'datalake_***' for task execution.
[2025-06-04T09:16:06.678+0000] {logging_mixin.py:188} INFO - ✅ Kết nối thành công. PostgreSQL version: ('PostgreSQL 16.9 (Ubuntu 16.9-0ubuntu0.24.04.1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0, 64-bit',)
[2025-06-04T09:16:06.695+0000] {logging_mixin.py:188} INFO - ✅ Tạo bảng thành công hoặc bảng đã tồn tại.
[2025-06-04T09:16:06.696+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-04T09:16:06.697+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-04T09:16:06.711+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_postgres_create_table_dag, task_id=create_postgres_table, run_id=manual__2025-06-04T09:16:05.437827+00:00, execution_date=20250604T091605, start_date=20250604T091606, end_date=20250604T091606
[2025-06-04T09:16:06.731+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-04T09:16:06.746+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-04T09:16:06.748+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
